npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/yarn_lockfile_updater.rb

```diff
@@ -171,9 +171,9 @@ def run_yarn_berry_subdependency_updater(yarn_lock:)
           update = "#{dep.name}@#{dep.version}"
 
           Helpers.run_yarn_commands(
-            "yarn add #{update}",
-            "yarn dedupe #{dep.name}",
-            "yarn remove #{dep.name}"
+            "yarn add #{update} --mode=update-lockfile",
+            "yarn dedupe #{dep.name} --mode=update-lockfile",
+            "yarn remove #{dep.name} --mode=update-lockfile"
```

Comment:
Opted to pass the argument vs automatically adding it in `run_yarn_commands`, since not all commands take that option and I figured it would lead to confusing errors down the line when using `run_yarn_commands` with one of those commands.

Reply:
Yeah, this seems like the right trade-off 👍🏻